app/views/shared/_breadcrumbs.html.erb

```diff
@@ -2,8 +2,8 @@
   <%= link_to 'Home', root_path %> /
 
   <%- parent_pages.each do |parent_page| -%>
-    <%= link_to parent_page[:name], parent_page[:path] %> /
+    <%= link_to raw(parent_page[:name]), parent_page[:path] %> /
   <%- end -%>
 
-  <%= page_title %>
+  <%= raw(page_title) %>
```

Comment:
The naming of this local variable, `page_title`, is confusing since we have a helper, `page_title`. But… it seems like Rails prefers the local variable passed to the partial instead of attempting to call the helper.

¯\\_(ツ)\_/¯

Reply:
Rails is magic. As in, shrouded in mystery, involving arcane incantations, and inflexible casting requirements.